portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,25 +31,105 @@
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> quotes;
-
-  @Override
-  public void init() {
-    quotes = new ArrayList<>(4);
-    quotes.add("Hope you are having a good one!");
-    quotes.add("How are you :)");
-    quotes.add("It is fantastic to see you!");
-    quotes.add("Thanks for vising my portfolio!");
-  }
+  private FeedbackRecord comment;
```

Comment:
Either rename the member variable `comment` to feedbackRecord  or rename the class FeedbackRecord to CommentRecord/UserComment. so it's easier to track what type it is

Reply:
Got it, thank you! I will make the change. I will substitute variables that contain "feedback" in the different source files.
For the pop-up window that tells the user "thank you for submitting feedback", I feel that "feedback" might be a better user-facing description here, so I might keep that one instance of "feedback"

Reply:
I've made the change to all the variables. Thank you for pointing this out! I feel like the code is more readable now.